### PR TITLE
Remove entry after delete event

### PR DESCRIPTION
**Pull request by jorgebay**
Ref: LOG-8681

### CODE REVIEW

**Comment by jorgebay** [comment]
I'm trying to come up w/ a integration test to cover it, I think I can have it before EOD.

**Review by c-nixon — APPROVED** [review]
Very nice, this is definitely part of the symlink issue from before. I look forward to seeing how you test this! 😄

File: Makefile [hunk 1]
```diff
@@ -99,8 +99,8 @@ test: test-journald ## Run unit tests
 	$(RUST_COMMAND) "--env RUST_BACKTRACE=full" "cargo test --no-run && cargo test"
 
 .PHONY:integration-test
-integration-test: ## Run integration tests
-	$(RUST_COMMAND) "--env LOGDNA_INGESTION_KEY=$(LOGDNA_INGESTION_KEY) --env LOGDNA_HOST=$(LOGDNA_HOST) --env RUST_BACKTRACE=full" "cargo test --manifest-path bin/Cargo.toml --features integration_tests -- --nocapture --test-threads=$(INTEGRATION_TEST_THREADS)"
+integration-test: ## Run integration tests using image with additional tools
+	$(DOCKER_JOURNALD_DISPATCH) "--env LOGDNA_INGESTION_KEY=$(LOGDNA_INGESTION_KEY) --env LOGDNA_HOST=$(LOGDNA_HOST) --env RUST_BACKTRACE=full" "cargo test --manifest-path bin/Cargo.toml --features integration_tests -- --nocapture --test-threads=$(INTEGRATION_TEST_THREADS)"
```
Comment (jorgebay) [inline Makefile:103]: I'll create a follow up ticket to rename the journald scripts and dockerfile folder to something like "dev"

**Comment by jorgebay** [comment]
I've pushed a test using `lsof` and verified that it fails against `master`.

File: docker/journald/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 FROM docker.io/logdna/build-images:rust-buster-stable
-RUN apt update -y && apt install -y systemd
+RUN apt update -y && apt install -y systemd lsof procps
```
Comment (c-nixon) [inline docker/journald/Dockerfile:2]: We should add these to the base image going forward

**Review by c-nixon — APPROVED** [review]
This is great

File: docker/journald/Dockerfile (hunk 2 above)
Comment (jorgebay) [reply docker/journald/Dockerfile:2]: oh, TIL about https://github.com/logdna/build-images

Good call, I'll move it there as part of the follow up ticket.

**Pull request merged**